packages/identity-kit/src/libs/react/provider.tsx

```diff
@@ -13,11 +13,18 @@ import { IdentityKitModal } from "./modal"
 import { IdentityKit } from "../../lib/identity-kit"
 import { GetSupportedStandardResponse, ICRC25Methods } from "../../standards/icrc-25"
 
+export type IdentityKitProviderTheme = "light" | "dark"
+
 interface IdentityKitProviderProps extends PropsWithChildren {
   signers: SignerConfig[]
+  theme?: IdentityKitProviderTheme
 }
 
-export const IdentityKitProvider: React.FC<IdentityKitProviderProps> = ({ children, signers }) => {
+export const IdentityKitProvider: React.FC<IdentityKitProviderProps> = ({
+  children,
+  signers,
+  theme = "light",
```

Comment:
I would double-check with Artem if we don't need any other configs. Just in case.
- rounded corners or not
- primary color
- secondary color
- maybe we want to pass backdrop as config?

Reply:
@anadiein do we need any of these customizations now?

Reply:
not sure about 'backdrop' (whatever it is (: ) 
all the rest points are not valid, confirmed with Dan and Egor previously.